aerogear/keycloak-metrics-spi
# Add keycloak online/offline sessions records

## Motivation
Add online/offline sessions by realm and clientId. Similar to #51 

## What
Add 2 Gauge : 
- keycloak_online_sessions
- keycloak_offline_sessions

## Why
Show the number of sessions

## How
On each user/admin events, put in a Gauge the result of the `getActiveClientSessionStats`.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Build the SPI from this branch and start Keycloak with it.
2. Login with users
2. Open the metrics endpoint in a browser. 


## Checklist:


## Progress


## Additional Notes
I'm a newby with Prometheus metrics, so fell free to comment and propose better implementation.

The  `getActiveClientSessionStats` contains only ClientId with active sessions. When all sessions of this client are logout, I don't found a solution to set the metric to `0.0`.
